### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by iam-hbk**

### CODE REVIEW

**Comment by KenanK0** [comment]
**Some import are incorrect and they prevent the code from running**

**in Profile**
`- import LeaderboardItem from "@/components/leaderboardItem"; 
`
instead of 
`- import LeaderboardItem from "@/components/LeaderboardItem";`


**in home header and coin Header**
`import CoinIconAndAmount from "./coinIconAndAmount";`
instead of
`import CoinIconAndAmount from "./CoinIconAndAmount";`

> Keep pushing

File: app/(app)/profile/index.tsx [hunk 1]
```diff
@@ -14,7 +14,8 @@ import { Stack } from "@rneui/layout";
 import { selectUser, unSetUser } from "@/utils/redux/features/user/userSlice";
 import { useDispatch, useSelector } from "react-redux";
 import { PaveCoins, ActivityIcon, TrophyIcon } from "@/components/icons";
-import LeaderboardItem from "@/components/LeaderboardItem";
+import LeaderboardItem from "@/components/leaderboardItem";
+
```
Comment (KenanK0) [inline app/(app)/profile/index.tsx:18]: comme ca vraiment

File: components/homeHeader.tsx [hunk 2]
```diff
@@ -15,88 +14,30 @@ import {
...
 import { DrawerHeaderProps } from "@react-navigation/drawer";
 import themeColors from "@/assets/colors";
-import { BlurView } from "expo-blur";
 import { router } from "expo-router";
-import axios from "axios";
-import CoinIconAndAmount from "./CoinIconAndAmount";
-import Toast from "react-native-toast-message";
-import { useSelector } from "react-redux";
-import { selectUser } from "@/utils/redux/features/user/userSlice";
+import CoinIconAndAmount from "./coinIconAndAmount";
```
Comment (KenanK0) [inline components/homeHeader.tsx:18]: Merci

File: components/coinHeader.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ import themeColors from "@/assets/colors";
 import { BlurView } from "expo-blur";
 import { router } from "expo-router";
 import { Entypo } from "@expo/vector-icons";
-import CoinIconAndAmount from "./CoinIconAndAmount";
+import CoinIconAndAmount from "./coinIconAndAmount";
```
Comment (KenanK0) [inline components/coinHeader.tsx:20]: donc tu as seulement changer l'import

**Review by KenanK0 — CHANGES_REQUESTED** [review]

File: components/homeHeader.tsx (hunk 2 above)
Comment (iam-hbk) [reply components/homeHeader.tsx:18]: Eish !😂😂

**Review by KenanK0 — APPROVED** [review]
good

**Pull request merged**